packages/decoder/test/current/test/downgrade-test.js

```diff
@@ -295,7 +302,9 @@ describe("Graceful degradation when information is missing", function () {
     assert.strictEqual(decodedFunction.value.selector, selector);
   });
 
-  it("Decodes transactions sent to the instance via additionalContexts", async function () {
+  //todo web3js-migration error withch schema
+  // Error: schema with key or id "5f79e70fd2b8a3a84f947e1f2376b6311f756dc26ed9ebfe148d5b0ef3350ac3f933c17a46bbf7fa08c2edb8a39381688f05ed60ff4853d4a72ea3fed8a85b58" already exists
```

Comment:
It now works after the updates to the validation function that is available in RC1.
However, the timeout needs to be extended. And this could be done for example by calling inside the test function: `this.timeout(3000);`. Or by calling on the test function itself like in:
```ts
it("...", async function () {
    ...
}).timeout(3000);
```